pkg/cli/manifest/parser.go

```diff
@@ -0,0 +1,199 @@
+/*
+Copyright 2023 The Radius Authors.
```

Comment:
should this be 2024 :)

Reply:
If you want to change our style/practices please start a discussion outside of this PR.